frontend/src/components/nodes/Nodes.tsx

```diff
@@ -1,36 +1,27 @@
 import React from "react";
 
-import NodesApi, * as N from "../../libraries/explorer-wamp/nodes";
+import * as N from "../../libraries/explorer-wamp/nodes";
+import { SubConsumer } from "../../context/SubscriptionProvider";
 
 import NodeRow from "./NodeRow";
 import PaginationSpinner from "../utils/PaginationSpinner";
 
-interface State {
-  onlineNodes?: N.NodeInfo[];
-}
-export default class extends React.Component<State> {
-  state: State = {};
-
-  getNodes = async () => {
-    let onlineNodes = await new NodesApi().getOnlineNodes();
-    this.setState({ onlineNodes });
-  };
-
-  componentDidMount() {
-    this.getNodes();
-  }
-
+export default class extends React.Component {
   render() {
-    const { onlineNodes } = this.state;
-    let nodes;
-    if (onlineNodes) {
-      nodes = onlineNodes.map((node: N.NodeInfo) => (
-        <NodeRow key={node.nodeId} node={node} />
-      ));
-    }
-    if (!nodes) {
-      return <PaginationSpinner hidden={false} />;
-    }
-    return <>{nodes}</>;
+    return (
+      <SubConsumer>
+        {(context) => (
+          <>
+            {context.nodeInfo.onlineNodes ? (
```

Comment:
If I open a Nodes list page, do I need to wait 10 seconds to see any data? That is suboptimal.